commands/enrich/enrich.go

```diff
@@ -78,26 +82,26 @@ func AppendVulnsToJson(results *utils.Results) error {
 		return err
 	}
 	var vulnerabilities []map[string]string
-	xrayResults := results.GetScaScansXrayResults()[0]
+	xrayResults := cmdResults.GetScaScansXrayResults()[0]
```

Comment:
This call returns a slice, depends on the amount of Targets in cmdResults. Are we sure this number cannot be 0?
if it does the slice is empty and we will get a nil pointer error